docs/BR.md

```diff
@@ -1008,6 +1063,62 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
 
 CAs MUST document potential issuances that were prevented by a CAA record in sufficient detail to provide feedback to the CAB Forum on the circumstances, and SHOULD dispatch reports of such issuance requests to the contact(s) stipulated in the CAA iodef record(s), if present. CAs are not expected to support URL schemes in the iodef record other than mailto: or https:.
 
+#### 3.2.2.9 Multi-Perspective Issuance Corroboration
+
+Multi-Perspective Issuance Corroboration attempts to corroborate the determinations made by the Primary Network Perspective from multiple additional Network Perspectives before subscriber certificate issuance. This process can improve protection against equally-specific prefix Border Gateway Protocol (BGP) attacks or hijacks.
+
+Each corroborating Network Perspective MUST independently verify: 
+- the presence of the expected 1) Random Value, 2) Request Token, 3) IP Address, or 4) Contact Address, as required by the relied upon validation method specified in Sections 3.2.2.4 and 3.2.2.5, and
+- the CA's authority to issue to the requested domain(s) or IP address(es), as specified in Section 3.2.2.8
+
+[Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control) and [Section 3.2.2.5](#3225-authentication-for-an-ip-address) describe the validation methods that require the use of Multi-Perspective Issuance Corroboration and how a Network Perspective can corroborate the outcomes determined by the Primary Network Perspective. A CA MAY use corroborating evidence for CAA record compliance for a period equal to the period of validity for Validation of Domain Authorization or Control specified in [Section 4.2.1](#421-performing-identification-and-authentication-functions).
+
+Results or information obtained from one Network Perspective MUST NOT be reused or cached when performing validation through subsequent Network Perspectives (e.g., different Network Perspectives cannot rely on a shared DNS cache to prevent an adversary with control of traffic from one Network Perspective from poisoning the DNS cache used by other Network Perspectives). The network infrastructure providing Internet connectivity to a Network Perspective MAY be administered by the same organization providing the computational services required to operate the Network Perspective.
+
+CAs MAY immediately retry Multi-Perspective Issuance Corroboration using the same validation method or an alternative method (e.g., a CA can immediately retry validation using "Email to DNS TXT Contact" if "Agreed-Upon Change to Website - ACME" does not corroborate the outcome of Multi-Perspective Issuance Corroboration). There is no stipulation regarding the maximum number of validation attempts that may be performed in any period of time.
```

Comment:
Question: does "[retrying] Multi-Perspective Issuance Corroboration" mean retrying the entire process (Primary Network Perspective and all remote perspectives) or could it mean just retrying the perspectives that have failed?  I think it should mean the former but I think this text could be clearer on this point.

Perhaps this:
```suggestion
CAs MAY immediately retry Multi-Perspective Issuance Corroboration, checking all Network Perspectives, using the same validation method or an alternative method (e.g., a CA can immediately retry validation using "Email to DNS TXT Contact" if "Agreed-Upon Change to Website - ACME" does not corroborate the outcome of Multi-Perspective Issuance Corroboration). There is no stipulation regarding the maximum number of validation attempts that may be performed in any period of time.
```

Reply:
Good point, @robplee. 

IMO, we should also avoid "locking in" CAs from reusing the same set of perspectives that resulted in the failure (e.g., failure may have happened because one of the perspectives is down and can't even attempt to corroborate). Our primary objective is ensuring that there's no "reuse" of corroborations across attempts.

What do you think of [this](https://github.com/ryancdickson/staging/commit/b5b3c5afc6af4f648ea72bdee3cd535fc000edbd) change?

Reply:
I think that change is excellent and perfectly addresses my concern.  I agree on not being locked into perspectives, the main thing I wanted to avoid was a situation where 1 perspective passes and 3 fail so then the server tries three perspectives and now only 2 fail etc as then the BGP issues may still be there, they just involve a bit more work and a bit less timing!